### ISSUE TITLE
How is the data set divided?

### ISSUE THREAD

**Issue by cookie-ke** [weihaox/Multi-Modal-CelebA-HQ#6]
Thanks a lot for your work! How are the train set and test set divided in the dataset? The odd thing is that the filename of the test set I see in the debug state is distributed between 0 and 23999. [In your previous response](https://github.com/IIGROUP/TediGAN/issues/16), you said the train set contains 0-23999 and the rest is the test set.
![image](https://user-images.githubusercontent.com/56613566/124207857-f6212f00-db18-11eb-828a-20c3c1f2a533.png)
![image](https://user-images.githubusercontent.com/56613566/124208154-911a0900-db19-11eb-868e-f328150ecd16.png)

**Comment by weihaox** [comment]
The filenames of the training set are from 0 to 23999. The names of the pickle files that contain training and test images are the same. They are chosen by the folder name: `test` or `traning`. Please make sure you choose the right mode. If the mode is correct, it must be my mistake to upload the files to the wrong paths. Please switch them into the right paths when using the dataset.

**Comment by cookie-ke** [comment]
Thank you so much！
